lab-guides/01-AzureLocalCloudDeployment/readme.md

```diff
@@ -58,16 +58,16 @@ Dell only supports Dell ISO that includes Dell drivers.
 
 * Make sure you hydrate latest Azure Local Image using CreateParentDisk.ps1 located in ParentDisks folder
 
-* Note: this lab uses ~60GB RAM. To reduce amount of RAM, you would need to reduce number of nodes.
+* Note: this lab uses ~76GB RAM. To reduce amount of RAM, you would need to reduce number of nodes.
 
 ### LabConfig
 
 ```PowerShell
 $LabConfig=@{AllowedVLANs="1-10,711-719" ; DomainAdminName='LabAdmin'; AdminPassword='LS1setup!' ; DCEdition='4'; Internet=$true; TelemetryLevel='Full' ; TelemetryNickname='' ; AdditionalNetworksConfig=@(); VMs=@()}
 
-#Azure Local 23H2
+#Azure Local 24H2
 #labconfig will not domain join VMs
-1..2 | ForEach-Object {$LABConfig.VMs += @{ VMName = "ALNode$_" ; Configuration = 'S2D' ; ParentVHD = 'AzSHCI24H2_G2.vhdx' ; HDDNumber = 4 ; HDDSize= 1TB ; MemoryStartupBytes= 24GB; VMProcessorCount="MAX" ; vTPM=$true ; Unattend="NoDjoin" ; NestedVirt=$true }}
```

Comment:
as per documentation for Cloud App Deployment the requirements are set to 32GB per node now.
additionally found in https://learn.microsoft.com/en-us/azure/azure-local/concepts/system-requirements-23h2?view=azloc-2507&tabs=azure-public

Reply:
32gb is official, but it works well with less. I successfully tested with 24, even with 16. It's important for people who does not have host with large amount of memory.

Then we should change max calculation instead?

Reply:
Hi Jaromir, the MAX calculation should be changed. see https://github.com/microsoft/MSLab/issues/635
It causes very often issues in my mslab but also on other machines.
You can leave the 24GB then, but please add a remark why. I hope they will not enforce the 32GB RAM in a HW req. check.

Reply:
@machv we need to modify calculation :). 
<img width="846" height="362" alt="image" src="https://github.com/user-attachments/assets/7351a180-ca4f-42e3-a2ba-74cf0a0b86be" />

Reply:
I fixed it in latest push in dev in MSLab